src/ElasticSearch.php

```diff
@@ -257,7 +236,7 @@ protected function buildResultSet($elasticResult = [])
     protected function updateIfUri($query)
     {
         if (\common_Utils::isUri($query)) {
-            $query = '"'.$query.'"';
+            $query = '"' . $query . '"';
```

Comment:
I know that it was not you, but it would be nice to add the return type hint :) on the method.